pkgs/firehose/lib/src/utils.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+/// Execute the given CLI command asynchronously, streaming stdout and stderr to
+/// the console.
+///
+/// This will also echo the command being run to stdout and indent the processes
+/// output slightly.
+Future<int> stream(
```

Comment:
Do you have a better name than `stream`, which is rather unspecific? `runCommand` maybe?

Reply:
+1 - I'll rename to something less overloaded